sei/chain.json

```diff
@@ -0,0 +1,118 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "sei",
+    "status": "live",
+    "website": "https://www.sei.io/",
+    "network_type": "mainnet",
+    "pretty_name": "Sei",
+    "chain_id": "pacific-1",
+    "bech32_prefix": "sei",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Ugh I always forget that part. Thank you. 

https://github.com/satoshilabs/slips/pull/1584